packages/vite/src/node/plugins/css.ts

```diff
@@ -471,7 +478,9 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
             )
           }
         }
+        const removedPureCssFiles = removedPureCssFilesCache.get(config)
         pureCssChunks.forEach((fileName) => {
+          removedPureCssFiles!.set(fileName, bundle[fileName] as RenderedChunk)
```

Comment:
Just a detail, but I think that it is better to assert that we know the map is not empty in 481. IMO it communicates better the intention, and we may later need to use `removedPureCssFiles` again.
```suggestion
        const removedPureCssFiles = removedPureCssFilesCache.get(config)!
        pureCssChunks.forEach((fileName) => {
          removedPureCssFiles.set(fileName, bundle[fileName] as RenderedChunk)
```